### PR TITLE
migrate WPF ControlsGallery to .NET Core

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
### Issues Resolved ### 

- continue migration:
https://github.com/xamarin/Xamarin.Forms/issues/7885
https://github.com/xamarin/Xamarin.Forms/issues/5092

### API Changes ###

 
 None

### Platforms Affected ### 

None

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###

- [x] Targets the correct branch
- [x] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Xamarin.Forms.ControlGallery.WPF/Xamarin.Forms.ControlGallery.WPF.csproj [hunk 1]
```diff
@@ -36,111 +34,36 @@
...
-    <Reference Include="System.Xaml">
-      <RequiredTargetFramework>4.0</RequiredTargetFramework>
-    </Reference>
-    <Reference Include="WindowsBase" />
-    <Reference Include="PresentationCore" />
-    <Reference Include="PresentationFramework" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Meziantou.WpfFontAwesome" Version="4.7.0" />
+    <PackageReference Include="Meziantou.WpfFontAwesome" Version="5.13.0" />
```
Comment (VladislavAntonyuk) [inline Xamarin.Forms.ControlGallery.WPF/Xamarin.Forms.ControlGallery.WPF.csproj:37]: version 5.13.0 supports netcore

**Comment by VladislavAntonyuk** [comment]
@rmarinho isn't is an issue with CI? I can manually build it without issue.

**Comment by rmarinho** [comment]
Doesn't seem so s, did you `git clean` and tried to build ?

**Comment by VladislavAntonyuk** [comment]
I reproduced it. It looks weird. It says the project has dependencies from net40, but we use net461. Maybe some packages were built with target framework net40. I will take a look

**Comment by VladislavAntonyuk** [comment]
@rmarinho, I committed, but didn't push final changes. it should work now

File: Xamarin.Forms.Maps.WPF/Xamarin.Forms.Maps.WPF.csproj [hunk 2]
```diff
@@ -1,99 +1,20 @@
...
-    <FileAlignment>512</FileAlignment>
-    <ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <WarningLevel>4</WarningLevel>
-    <TargetFrameworkProfile />
-    <!-- 
-        This is needed for non SDK style project so the IDE will pick the correct LangVersion.
-        Once this is an SDK style project it will pick this up from the Directory.Build.Props file correctly
-     -->
-    <LangVersion>7.3</LangVersion>
+    <TargetFramework>net461</TargetFramework>
```
Comment (VladislavAntonyuk) [inline Xamarin.Forms.Maps.WPF/Xamarin.Forms.Maps.WPF.csproj:3]: we have a dependency on System.Device which is not ported to .NET Core. so build only net461

File: Xamarin.Forms.Platform.WPF/Microsoft.Windows.Shell/WindowChrome.cs [hunk 3]
```diff
@@ -367,12 +367,8 @@ public WindowChrome()
                     bp.DependencyProperty,
                     new Binding
                     {
-#if NET4_5
```
Comment (VladislavAntonyuk) [inline Xamarin.Forms.Platform.WPF/Microsoft.Windows.Shell/WindowChrome.cs:370]: it is always false, so just removed

**Comment by rmarinho** [comment]
Ok new failure
https://dev.azure.com/xamarin/public/_build/results?buildId=20935&view=logs&j=b3f59cb3-a6e8-503e-b7b8-e24afbf6cb44&t=32b10b9b-8ebe-5a3f-0358-c557bd2ca7d3&l=117

```
Attempting to build package from 'Xamarin.Forms.Maps.WPF.nuspec'.
Error NU5019: File not found: '..\Xamarin.Forms.Maps.WPF\bin\Debug\Xamarin.Forms.Maps.WPF.dll'.
```

**Comment by VladislavAntonyuk** [comment]
@rmarinho  please try again. fixed path

**Pull request merged**